helpdesk_mgmt/models/helpdesk_ticket.py

```diff
@@ -213,7 +213,7 @@ def message_new(self, msg, custom_values=None):
             )
             if p
         ]
-        ticket.message_subscribe(partner_ids)
+        ticket.message_subscribe(partner_ids.ids)
```

Comment:
This is not correct, as `partner_ids` is a list, not a recordset. You should change previous line and put `[p.id for ...]` instead.